CDP4WebServices.API/Modules/Authentication/AuthenticationModule.cs

```diff
@@ -34,7 +34,8 @@ public AuthenticationModule(ICDP4WebServiceAuthentication webServiceAuthenticati
                     }
 
                     // Identify user's location and save this data
-                    this.ContributorLocationResolver.GetLocationDataAndSave();
+                    // TODO revise geo ip service, it does not work properly, that is why it is commented
```

Comment:
please add a ticket on phabricator and put reference to it here, otherwise we will forget

Reply:
I created an issue for this. Maybe insert issue number from GitHub?

Reply:
> 
> 
> I created an issue for this. Maybe insert issue number from GitHub?

since the webapp is developed on phab for now, no GH issue please